src/class-wpml-beaver-builder-translatable-nodes.php

```diff
@@ -68,16 +88,23 @@ public function update( $node_id, $settings, WPML_PB_String $string ) {
 			if ( $this->conditions_ok( $node_data, $settings ) ) {
 				foreach ( $node_data['fields'] as $field ) {
 					$field_key = $field['field'];
-					if ( $this->get_string_name( $node_id, $field, $settings ) == $string->get_name() ) {
+					if ( $this->get_string_name( $node_id, $field, $settings ) === $string->get_name() ) {
 						$settings->$field_key = $string->get_value();
 					}
 				}
 				if ( isset( $node_data['integration-class'] ) ) {
 					try {
+						/**
+						 * Node object.
+						 *
+						 * @var WPML_Beaver_Builder_Module_With_Items $node
+						 */
 						$node = new $node_data['integration-class']();
 						$node->update( $node_id, $settings, $string );
+						// phpcs:disable Generic.CodeAnalysis.EmptyStatement.DetectedCatch
 					} catch ( Exception $e ) {
-
+						// Nothing to do with the exception, we do not handle it.
+						// phpcs:enable
```

Comment:
@kagg-design shouldn't be `// phpcs:enable Generic.CodeAnalysis.EmptyStatement.DetectedCatch`?

Reply:
@andreasciamanna it is on line 104, before `catch (`

Reply:
@kagg-design please don't close a conversation with no valid reason.

I can obviously see what's in line 104, but line 104 is for **disabling** the rule.  
The line I'm commenting is for **enabling** a rule.  
When you disable a rule, you should re-enable it by specifying the name of that rule.

If you disable 3 rules, how phpcs can know which one you are re-enabling?

Reply:
@andreasciamanna Unfortunately, I didn't understand you at first time. Now I have made required changes.